src/components/subsetDraft/Classification.jsx

```diff
@@ -154,38 +154,63 @@ export const Classification = ({item = {}, update, remove, from, to}) => {
 export const Codes = ({from, to, codes = [], id, include}) => {
     const {t} = useTranslation();
 
+    const codesToLoad = 35; // This is how many codes we load on first render. The rest can wait.
+
+    // React state objects and setters
+    const [sFrom, setFrom] = useState(from);
+    const [sTo, setTo] = useState(to);
+    const [sAllCodes, setAllCodes] = useState(codes);
+    const [sRenderedCodes, setRenderedCodes] = useState(sAllCodes.slice(0, Math.min(codesToLoad, sAllCodes.length)));
+    const [sId, setId] = useState(id);
+    const [sInclude, setInclude] = useState(include);
```

Comment:
Code included/excluded state is already kept and handeled in AppContext subset variable. No need to keep it localy in the object. It is confusing. 

**All state change has to happen throught `AppContext`!** 

For the subset draft you have to use reference to the `AppContext`.`subset` variable. This variable uses useReduser react hook which returns `{draft, dispatch}` object. Use `draft` to pick up data, and `dispatch()` to fire state actions and provide relevant user input to the AppContext. The AppContext (in this case `subset.dispatch()`) will decide how to handle the input and how it will affect the state. 
:)

Reply:
Ok, this is hard for me to grapple with: If I get the AppContext and change the codes list in the subset variable to be only the 35 first items so that only those items will display, doesn't that mean I will remove relevant items and not get the full list handed to me in the Codes component on the next update, and thus not be able to display the whole list on the next frame? I tested this, and it seems to be the case. So is my approach wrong on a deeper level? I don't think I understand what is the "reacty" way of doing this, without making this component "smart".